### PR TITLE
 Change Signup and Login views

### PR DESCRIPTION
**Pull request by jparrax**
Change in login the element from <button> to <a>, To simplify the logic,
because with button you have to add more code.

Login's place holder was changed in signup.php and login.php to be more
user friendly.

### CODE REVIEW

File: view/footer.php [hunk 1]
```diff
@@ -7,29 +7,19 @@
...
             </ul>
         </div>
         <div class="col-6 col-md">
             <h5>About</h5>
             <ul class="list-unstyled text-small">
                 <li><a class="text-muted" href="#">Team</a></li>
-                <li><a class="text-muted" href="#">Locations</a></li>
-                <li><a class="text-muted" href="#">Privacy</a></li>
-                <li><a class="text-muted" href="#">Terms</a></li>
+                <li><a class="text-muted" href="#">Terms</a></li>
```
Comment (AntonRozhanskii) [inline view/footer.php]: Why do we need this line?

Comment (jparrax) [reply view/footer.php]: Sorry, it was in Fernando's branch.

Comment (jparrax) [reply view/footer.php]: You should delete it, We don't it yet.

**Review by AntonRozhanskii — COMMENTED** [review]
Your master's version is deprecated.
Pull actual master branch first.
Probably, you will need to solve conflicts

**Pull request closed without merging**